packages/mimic-thrift/src/provider.ts

```diff
@@ -264,9 +265,12 @@ export class ThriftProvider extends EventEmitter implements IServiceProvider, IC
         });
       } else {
         const { data, exception } = this.respManager.find(id)[func.name] || {
-          data: generateThriftResponse(func.returnTypeId, def, func.returnType, func.returnExtra),
+          data: new Thrift.TApplicationException(
+            Thrift.TApplicationExceptionType.MISSING_RESULT,
+            `Mimic: no method found named:${func.name}`),
```

Comment:
could add a space between `named:` and `${func.name}` but nbd

Reply:
I would also change a text to `Mimic: no data found for function ${func.name}`